Library/Formula/strider.rb

```diff
@@ -0,0 +1,93 @@
+class UniversalNpm < Requirement
+  fatal true
+  satisfy { which("npm") }
+  def message
+    "npm is required. If you have installed node with `--without-npm` option, reinstall with `--with-npm`."
+  end
+end
+
+class UniversalNode < Requirement
```

Comment:
This syntax is already used - I copied it from the formula for skinny. See https://github.com/Homebrew/homebrew/blob/master/Library/Formula/skinny.rb